### DISCUSSION TITLE
When will the next release come out?

### DISCUSSION THREAD

**Discussion by oehme** [grafana/pyroscope#1964] [General]
The last release is from February and there were some huge table rendering performance improvements since then. These take rendering the table view from several seconds to almost instantaneous. It would be great if a release with these changes came out soon.

**Comment by Rperry2174** [comment]
hi @oehme thanks for reaching out and sorry for the issue here. The reason we haven't cut a new release is because of our license change from Apache 2 to aGPL. We will be soon (aiming for mid-august ) merging Pyroscope with the Phlare project and releasing it as Pyroscope 1.0. 

This way it makes it easier for the community to know that everything _before_ Pyroscope 1.0 is Apache 2 and everything after is aGPL.

**Comment by oehme** [comment]
Thank you for the rough timeline, looking forward to 1.0 :slightly_smiling_face:

**Comment by korniltsev** [comment]
if you are brave enough you can use builds from main branch :)

https://hub.docker.com/r/pyroscope/pyroscope-dev

**Reply by oehme** [reply]
Oh cool, I had missed those. Thanks!
